ui/src/components/friends/add.rs

```diff
@@ -89,12 +102,19 @@ pub fn AddFriend(cx: Scope) -> Element {
                         | Error::BlockedByUser
                         | Error::InvalidIdentifierCondition
```

Comment:
I agree that some of the match arms can be deleted so that the error would fall through to the  `_ => ` arm. It might be helpful to have a custom toast to notify the user that they are blocked.

Reply:
I could see some people not wanting someone to know that they are blocked though right? Like some stalker checking to see if someone is on the app or something. Maybe make it more vague, idk?